src/pages/partners/payment-agent/_p2p_banner.tsx

```diff
@@ -108,7 +107,6 @@ const P2PBanner = () => {
                     <LeftChild>
                         <StyledHeader
                             as="h3"
-                            type="heading-3"
```

Comment:
I should remove this because in 75% and below zoom looks as 40px and looks so bad